app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,3 @@
+<footer>
+  <p>© BrightCoders Equipo 35. Todos los derechos reservados.</p>
+</footer>
```

Comment:
add a blank line at the end of this file too.

Reply:
done  :+1: